packages/components/dropdown/src/Dropdown.stories.tsx

```diff
@@ -34,6 +36,67 @@ export const Default: StoryFn = _args => {
   )
 }
 
+export const Controlled: StoryFn = () => {
+  const [value, setValue] = useState('book-1')
+
+  return (
+    <div className="w-sz-480 pb-[300px]">
+      <Dropdown value={value} onValueChange={setValue}>
```

Comment:
The value is not handled by the Dropdown.value component?

Reply:
Actually no :) We read the Radix doc too fast last time. `Dropdown.Value` is just for the display of the current selected value inside the trigger, but it does not manage it.

All the main props are on the root element: https://www.radix-ui.com/primitives/docs/components/select#root

And this is also how we did for other components until now:
![Uploading Capture d’écran 2023-11-29 à 11.31.20.png…]()